tests/testthat/_snaps/bump_version/NEWS2.md

```diff
@@ -1,4 +1,4 @@
-<!-- NEWS.md is maintained by https://cynkra.github.io/fledge, do not edit -->
+<!-- NEWS.md is maintained by https://cynkra.github.io/fledge/, do not edit -->
```

Comment:
The terminal slash is new, and adds noise...

Reply:
fixed